samples/SoundBoard/src/main/java/com/google/oboe/samples/soundboard/MusicTileView.java

```diff
@@ -20,16 +36,41 @@
     private boolean[] mIsPressedPerRectangle;
     private Paint mPaint;
     private SparseArray<PointF> mLocationsOfFingers;
-    private NoteListener mNoteListener;
+    private TileListener mNoteListener;
+
+    public interface TileListener {
+        public void onTileOn(int index);
+        public void onTileOff(int index);
+    }
+
+    public class NoteListener implements TileListener {
```

Comment:
This goes in its own file like before. Sorry that I was not clear.

Reply:
Thanks for going over this with me offline. Done!